service/src/main/java/com/fecklessweasel/service/datatier/CourseTable.java

```diff
@@ -46,4 +46,22 @@ public static int insertCourse(Connection conn, int univid, int deptid, int cour
             throw new ServiceException(ServiceStatus.DATABASE_ERROR, ex);
         }
     }
+    /**
+     *@author Hayden Schmackpfeffer
+     *@param conn SQL Connection
+     *@param deptid Department ID that all returned courses will have
+     *@return all tuples in Course table where deptid= deptid
+     */
+    public static ResultSet getAllWithDept(Connection conn, int deptid) throws ServiceException {
```

Comment:
That is assuming that it's ok if the department object model calls the course data model, which I think it is.